transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportConnectorFactory.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.log.Logger;
+import io.trino.spi.connector.Connector;
+import io.trino.spi.connector.ConnectorContext;
+import io.trino.spi.connector.ConnectorFactory;
+import io.trino.spi.function.FunctionId;
+import java.io.File;
+import java.io.FileFilter;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.*;
+
+
+public class TransportConnectorFactory implements ConnectorFactory {
+
+  private static final String DEFAULT_TRANSPORT_UDF_REPO = "/transport-udf-repo";
+  private static final String TRANSPORT_UDF_REPO_CONFIG_NAME = "transport.udf.repo";
+  private static final Logger log = Logger.get(TransportConnectorFactory.class);
+
+  private static final FileFilter TRANSPORT_UDF_JAR_FILTER = (file) -> {
+    return file.isFile() && file.getName().endsWith(".jar") && !file.getName().startsWith("transportable-udfs");
```

Comment:
Can be simplified to be: `(file) -> file.isFile() && file.getName().endsWith(".jar") && !file.getName().startsWith("transportable-udfs");`

Reply:
Shall we be more specific with the exclusion here? This would prevent users from using the example UDFs. It seems  Trino only depends on the following Transport jars.

```
transportable-udfs-api-0.0.111.jar
transportable-udfs-trino-0.0.111.jar
transportable-udfs-type-system-0.0.111.jar
transportable-udfs-utils-0.0.111.jar
```